drake/examples/Quadrotor/runDynamics.cpp

```diff
@@ -16,8 +16,22 @@ using namespace Eigen;
 int main(int argc, char* argv[]) {
   const size_t num_lidar_points = 100;
 
+  // Get the final time of the simulation
```

Comment:
Missing punctuation (I will abbreviate this as _punc_).
https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar